src/app/app.module.ts

```diff
@@ -42,7 +43,9 @@ Amplify.configure(AWS_CONFIG);
       hotJarSv: environment.hotJarSv,
       enabled: environment.production
     }),
-    NgxsModule.forRoot([], { developmentMode: !environment.production }),
+    NgxsModule.forRoot([AdminsState], {
```

Comment:
Is it better forRoot here or in it's own module? looking more like an auth-module may be good to encapsulate all auth-related services, states, functions? just a thought though.